clang-10 warning: definition of implicit copy constructor is deprecated

In boost/phoenix/core/actor.hpp:
```
include/boost/phoenix/core/actor.hpp:148:9: error: definition of implicit copy constructor for 'actor<boost::log::v2s_mt_posix::expressions::aux::unary_function_terminal<boost::log::v2s_mt_posix::expressions::has_attribute<void> > >' is deprecated because it has a user-declared copy assignment operator [-Werror,-Wdeprecated-copy]
        BOOST_PROTO_EXTENDS_ASSIGN()
        ^
```

--- a/include/boost/phoenix/core/actor.hpp
+++ b/include/boost/phoenix/core/actor.hpp
@@ -145,7 +145,7 @@ namespace boost { namespace phoenix
 
         BOOST_PROTO_BASIC_EXTENDS(expr_type, actor<Expr>, phoenix_domain)
         BOOST_PROTO_EXTENDS_SUBSCRIPT()
-        BOOST_PROTO_EXTENDS_ASSIGN()
+        BOOST_PROTO_EXTENDS_ASSIGN_()
 
         template <typename Sig>
         struct result;
@@ -224,6 +224,8 @@ namespace boost { namespace phoenix
             return phoenix::eval(*this, phoenix::context(env, default_actions()));
         }
 #endif
+
+        BOOST_DELETED_FUNCTION(actor& operator=(actor const&))
     };
 }}
 

